open-metadata-implementation/access-services/asset-catalog/asset-catalog-client/docs/user/asset-catalog-omas-client.md

```diff
@@ -48,7 +48,9 @@ Once you have an instance of the client, you can use it to:
 
 * Returns the sub-graph that represents the list of assets that in neighborhood of the given asset - [getAssetsFromNeighborhood]()
 
-* Return a list of assets matching the search criteria without the full context [searchByType]()
+* Return a list of assets matching the search criteria without the full context [searchByType](). Entity types are case 
+insensitive, so proper values should be given in the request, otherwise an EntityNotKnownException will be thrown. If
```

Comment:
how does the user know what a proper value is? Can we add checking client side to as well to reject the request before it reaches the server. Maybe the client gets the supported types and then checks against those.

Reply:
I added details to the user documentation about the way to find out the supported types.

Reply:
@alexandra-bucur it would be better if the client could issue a call to the omas to get a list of supported types, the server side OMAS can expose any new types in a controlled way.  I recommend that this list also include any child types. For example if you support Topic you should also support KafkaTopic and any custom child types that have been added. I suggest this is not an enumeration as it could change in time; it should probably return a sanitised OMAS friendly version of the typedefs in question. The caller can then be 'metadata-driven' and use the attribute definitions of each type to inform its processing.

Reply:
@davidradl I though of this one to be the user documentation. Do you think that sentences added in this file are not enough?